persistence-api/src/main/java/org/apache/cassandra/stargate/transport/ProtocolVersion.java

```diff
@@ -18,11 +18,7 @@
 
 package org.apache.cassandra.stargate.transport;
 
-import java.util.ArrayList;
```

Comment:
nit: In general, we don't do `import java.util.*;`

Reply:
It's IDE setting, you should set `100` to import with `*`

![image](https://user-images.githubusercontent.com/10600041/186688807-8158f501-3714-4185-aef4-9fc023fdb40c.png)